File: 프로그래머스/1/155652. 둘만의 암호/둘만의 암호.py
def solution(s, skip, index):
    answer = ''
    check = "abcdefghijklmnopqrstuvwxyz"
    
    for item in skip:
        check = check.replace(item, "")
    
    n = len(check)
    for i in range(len(s)):
        target_index = check.index(s[i])
        answer += check[(target_index + index) % n]
        
    return answer

File: 프로그래머스/1/155652. 둘만의 암호/둘만의 암호.js
function solution(s, skip, index) {
    const ref = "abcdefghijklmnopqrstuvwxyz".split("").filter((item) => !skip.includes(item));
    
    let answer = '';
    for (const item of s) {
        const targetIndex = ref.indexOf(item) + index;
        answer += ref[targetIndex % ref.length];
    }
    return answer;
}